tests/test_adjoint_jacobian.py

```diff
@@ -268,8 +268,8 @@ def test_Rot_gradient(self, stateprep, theta, dev):
         assert np.allclose(calculated_val, numeric_val, atol=tol, rtol=0)
 
     @pytest.mark.skipif(
-        device_name != "lightning.qubit",
```

Comment:
Are these implemented for GPU with MPI? if not do we have to skip them?

Reply:
Good point. Unit tests for mpi support live in the `mpitests/` dir. All tests in this dir are for non-MPI backends